ansible/vsonic_vtb

```diff
@@ -0,0 +1,228 @@
+all:
```

Comment:
why are we duplicating this veos_vtb file to vsonic_vtb? it looks the same as veos_vtb.

Reply:
Remove vsonic_vtb. Please review it.